scalafix/rules/src/main/scala/fix/CanBuildFrom.scala

```diff
@@ -4,6 +4,8 @@ import scalafix._
 import scalafix.util._
 import scala.meta._
 
+import scala.collection.mutable.{Set => MSet}
```

Comment:
The recommended practice is to import `scala.collection.mutable` and then write `mutable.Set` at use site.